assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -20,6 +20,7 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { useMount } from 'react-use';
```

Comment:
I know IB was written like this, but we should not change anything in the component because it should not contain business logic. All changes should be made in the `UserInputPreview` component which should prepare proper data and pass it down to this component.